modules/account-lib/src/coin/baseCoin/enum.ts

```diff
@@ -29,6 +29,10 @@ export enum TransactionType {
   StakingWithdraw,
   // Handle smart contract calls
   ContractCall,
+  // Wallet owner opt-in to receive this asset in his wallet
+  EnableAsset,
+  // Wallet owner opt-out to receive this asset in his wallet
+  DisableAsset,
```

Comment:
you can't disable an asset, you can disable a token
```suggestion
  DisableToken,
```

Reply:
We choose that word (Asset) because this type will be used by other coins. And it maybe use other types.

Reply:
Isn't `token` a generic name for non native assets? if not, you should find other name wich represents non native assets, because each coin is an asset and you can't enable or disable native assets

Reply:
In Algorand the token operations are called "Assets Transactions". They use the term ASA (Algorand Standard Asset). See https://developer.algorand.org/docs/get-details/asa/

I choose the "Asset" instead of "Token" because other networks can require to  opt-in to receive other types of assets (including tokens, native coins, or NFTs).

What do you think?